Drop deprecated removal pg type.

aiicore opened the pull request:
This is related with ceph/ceph#6505 Ceph-object-corpus is a subrepo of ceph and 6505 test will fail if this won't be merged before them.

Signed-off-by: Igor Podoski igor.podoski@ts.fujitsu.com

liewegas commented:
Instead of removing the type, we can use the backward_incompat mechanism to indicate that newer code doesn't understand the older encoded types.  Do a quick find in the tree to see other instances of this...

aiicore commented:
Did you mean - create file in archive/VERSION/forward_incompat directory? I don't see backward_incompat in ceph/src/test/encoding/readable.sh (btw. I found little bug + formatting there https://github.com/ceph/ceph/pull/6714 ) If I use this, it will ignore whole coll_t - normal pg types testing too, right? Another thing is that I don't know what VERSION should I put - in what ceph version my code will be merged.

aiicore commented:
Updated with new approach introduced in #6932

Commit:
Drop deprecated removal pg type.

Two testing objects from coll_t type need to be removed.

Those objects consists FORREMOVAL_ pg type which is deprecaded.

Signed-off-by: Igor Podoski <igor.podoski@ts.fujitsu.com>
